io.github.monsterbirdlimited.soscap.yaml

```diff
@@ -0,0 +1,81 @@
+---
+app-id: io.github.monsterbirdlimited.soscap
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: soscap
+finish-args:
+  - --filesystem=home
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --device=dri
+  - --talk-name=org.freedesktop.FileManager1
+  - --filesystem=xdg-videos
+  - --env=G_MESSAGES_DEBUG=none
+  - --env=GST_DEBUG=3
+
+
+cleanup:
+  - '/include'
+  - '/lib/pkgconfig'
+  - '/man'
+  - '/share/doc'
+  - '/share/gtk-doc'
+  - '/share/man'
+  - '/share/pkgconfig'
+  - '*.la'
+  - '*.a'
+
+modules:
+  
+  - name: soscap
+    buildsystem: simple
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/Monsterbirdlimited/soscap
+        commit: c58ffa8d785d090a7191a0467621cc56ec9b7222
+        tag: v0.9.1-alpha 
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\\d.]+)$
+          is-main-source: true
+    build-commands:
+      - install -Dm755 soscap.py ${FLATPAK_DEST}/bin/soscap
+      - install -Dm644 io.github.monsterbirdlimited.soscap.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 io.github.monsterbirdlimited.soscap.png ${FLATPAK_DEST}/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.png
+
+  - name: x264
+    config-opts:
+      - --enable-shared
+      - --disable-cli
+      - --system-libx264
+      - --enable-pic
+      - --enable-lto
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        branch: stable
+
+  - name: libx264
+    config-opts:
+      - --enable-shared
+      - --disable-cli
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        branch: stable
+
+
+  - name: gstreamer-plugins-ugly
+    buildsystem: meson
+    config-opts:
+      - -D=x264=enabled
```

Comment:
are you sure it's the right option?

Reply:
Yes. it is the only way it'll actually work. Without it it gives an error specifying x264enc module missing or not found. It was the only solution we could find that worked properly.

Reply:
I do not see `=x264` as a valid option. But I guess when the build pass the manifest validation we'll see how this fail. It should have failed locally already, or put out a warning.

Reply:
It was recommended to me by someone else.  I don't know enough to know why that made it work.

Reply:
Not sure how meson parses it either, but I'd guess it should be `-Dx264=enabled`. You would need to check it in the meson.build file.

Reply:
I looked it up before asking the question:

https://gitlab.freedesktop.org/gstreamer/gst-plugins-ugly/-/blob/1.18.6/meson_options.txt#L19

But then the runtime has 1.20 so it's the wrong branch.